vscode/microsoft-kiota/src/utilities/deep-linking.ts

```diff
@@ -4,7 +4,8 @@ import { getSanitizedString, parseGenerationLanguage, allGenerationLanguagesToSt
 import { createTemporaryFolder } from "./temporary-folder";
 
 export function isDeeplinkEnabled(deepLinkParams: Partial<IntegrationParams>): boolean {
-  return Object.keys(deepLinkParams).length > 0;
+  const minimumNumberOfParams = 1;
+  return Object.values(deepLinkParams).filter(property => property).length >= minimumNumberOfParams;
```

Comment:
Does this add any value or adds complexity?

Reply:
This checks whether the values of the properties of the integration parameters have values instead of whether the properties exist. 
After creating a plugin, the plugin name is in the search parameters when choosing a new OAD. and hence transferred to the deeplink parameters. Leading to a false deeplink scenario.
ideally deep links would have more than one value passed for better context so this safeguards against a false deep link situation